### PR TITLE
QB-7: Create Navbar

### PR DESCRIPTION
**Pull request by mehedihassandev**
Create navber section

## Video

https://github.com/mehedihassandev/quickbuy/assets/59695824/b0f4e220-ec75-4baa-9809-903e72bd9d64

### CODE REVIEW

File: src/app/components/Navbar/index.tsx [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+              onClose={handleDrawerClose}
+              sx={{
+                "& .MuiDrawer-paper": {
+                  width: "35%",
+                },
+              }}
+            >
+              <List>
+                {menus.map((menu) => (
+                  <div key={menu}>
```
Comment (sun-mit) [inline src/app/components/Navbar/index.tsx:111]: why use div ? isn't there any other way ?

File: src/app/components/Navbar/index.tsx [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+        sx={{
+          bgcolor: "white",
+          color: "black",
+          px: 2,
+          boxShadow: "none",
+        }}
+      >
+        <Toolbar disableGutters>
+          {/* QB-7: Responsive menu */}
+          <Adb sx={{ display: { xs: "none", md: "flex" }, mr: 1 }} />
```
Comment (sun-mit) [inline src/app/components/Navbar/index.tsx:72]: can you please explain about Adb ?

File: src/app/components/Navbar/index.tsx [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+              fontFamily: "monospace",
+              fontWeight: 700,
+              letterSpacing: ".3rem",
+              color: "inherit",
+            }}
+          >
+            LOGO
+          </Typography>
+
+          <Box
```
Comment (sun-mit) [inline src/app/components/Navbar/index.tsx:166]: i think the menu items should be right side more not in center.

File: src/app/components/Navbar/index.tsx (hunk 2 above)
Comment (mehedihassan3) [reply src/app/components/Navbar/index.tsx:72]: this is the icon

File: src/app/components/Navbar/index.tsx (hunk 1 above)
Comment (mehedihassandev) [reply src/app/components/Navbar/index.tsx:111]: actually here i want to use a fragment <></> and I need to use key that's why i use the div but now I will replace it with <Fregment>. bcz best practice is use empty fragment instant of empty div. empty div means without style

**Review by sun-mit — APPROVED** [review]

**Pull request merged**